Yosemite/Yosemite/PointOfSale/Items/POSProductOrVariationResolver.swift

```diff
@@ -0,0 +1,73 @@
+import Foundation
+import protocol Networking.ProductsRemoteProtocol
+import class WooFoundation.CurrencySettings
+
+struct POSProductOrVariationResolver {
```

Comment:
I don't love the name, but I wanted to convey that we need to do quite a bit of work to get a variation from a product. Bring on a local database for this.